.github/workflows/drivers.yml

```diff
@@ -1116,3 +1116,41 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_TEST_RESULTS_SECRET_ACCESS_KEY }}
         aws-region: ${{ vars.AWS_REGION }}
         trunk-api-token: ${{ secrets.TRUNK_API_TOKEN }}
+
+  # Dummy job generator to spit out fake runs for matrix jobs we skipped. See
+  # https://github.com/orgs/community/discussions/9141 and
+  # https://metaboat.slack.com/archives/C0669P4AF9N/p1732315370301059
+  be-driver-tests-skipped-stub:
+    needs: files-changed
+    if: |
+      !cancelled() &&
+      (github.event.pull_request.draft == true ||
+      needs.files-changed.outputs.backend_all == 'false')
+    strategy:
+      matrix:
+        job:
+          -  "Test Postgres 12.x (App DB Tests)"
+          -  "Test Postgres 12.x (Driver Tests)"
+          -  "Test Postgres Latest (App DB Tests)"
+          -  "Test Postgres Latest (Driver Tests)"
+          -  "be-tests-mariadb-10-2-ee (0)"
+          -  "be-tests-mariadb-10-2-ee (1)"
+          -  "be-tests-mariadb-10-2-ee (2)"
+          -  "be-tests-mariadb-10-2-ee (3)"
+          -  "be-tests-mariadb-latest-ee (0)"
+          -  "be-tests-mariadb-latest-ee (1)"
+          -  "be-tests-mariadb-latest-ee (2)"
+          -  "be-tests-mariadb-latest-ee (3)"
+          -  "be-tests-mysql-8-0-ee (0)"
+          -  "be-tests-mysql-8-0-ee (1)"
+          -  "be-tests-mysql-8-0-ee (2)"
+          -  "be-tests-mysql-8-0-ee (3)"
+          -  "be-tests-mysql-latest-ee (0)"
+          -  "be-tests-mysql-latest-ee (1)"
+          -  "be-tests-mysql-latest-ee (2)"
+          -  "be-tests-mysql-latest-ee (3)"
```

Comment:
I fear this could become annoying and error-prone to maintain. The nice thing about the e2e stubs is it uses the same matrix generator that gives us the actual testing matrix is also used to generate the stub matrix. There's some real potential for de-sync here, but I think it's fine to generate it if/when it becomes a problem.

Reply:
yeah my thinking is that unfortunately it's already annoying and error-prone to maintain since making jobs required is done by name in the GitHub repo settings for a particular branch. So at least by explicitly writing out the names here we can be sure we're matching the ones that we had to explicitly make required